content/en/docs/v3.6/learning/api.md

```diff
@@ -489,3 +491,4 @@ message LeaseKeepAliveResponse {
 [locks]: https://github.com/etcd-io/etcd/blob/main/client/v3/concurrency/mutex.go
 [mvcc]: https://en.wikipedia.org/wiki/Multiversion_concurrency_control
 [stm]: https://github.com/etcd-io/etcd/blob/main/client/v3/concurrency/stm.go
+[consistency]: ../api_guarantees/#isolation-level-and-consistency-of-replicas
```

Comment:
Please use full url otherwise link checker in the pipeline will fail.

Reply:
Hold that thought, I'm not certain it's the link, let me check a few things and report back.

Reply:
Ok so replicating locally here is what it is upset about:

```bash
htmltest started at 08:39:52 on public
========================================================================
docs/v3.6/learning/api/index.html
  hash does not exist --- docs/v3.6/learning/api/index.html --> ../api_guarantees/#isolation-level-and-consistenc
y-of-replicas
```